codegen/config/initialisms.go

```diff
@@ -69,7 +17,7 @@ type GoInitialismsConfig struct {
 // setInitialisms adjustes GetInitialisms based on its settings.
 func (i GoInitialismsConfig) setInitialisms() {
 	toUse := i.determineGoInitialisms()
-	GetInitialisms = func() map[string]bool {
+	templates.GetInitialisms = func() map[string]bool {
```

Comment:
Moved due to cyclic dep, only ref is in templates, so thought it fits there well